src/core/osmLayer.js

```diff
@@ -43,6 +43,7 @@ geo.osmLayer = function (arg) {
     m_numberOfCachedTiles = 0,
     m_tileCacheSize = 100,
     m_baseUrl = 'http://tile.openstreetmap.org/',
+    m_subdomains = 'abc',
```

Comment:
I didn't notice it was configurable via the options.  In that case, can you document the option in the class docstring?